driver-async/src/main/com/mongodb/async/client/gridfs/helpers/AsynchronousChannelHelper.java

```diff
@@ -63,6 +63,12 @@ public void failed(final Throwable exc, final Object attachment) {
                 });
             }
 
+            @Override
+            public void skip(final long bytesToSkip, final SingleResultCallback<Long> callback) {
```

Comment:
Does this have to skip too?

Reply:
As these are helpers / wrappers for reading information into GridFS, I don't see a usecase for skipping via the API. As such we don't break contract and report we skipped 0.